src/app/components/Calendar.tsx

```diff
@@ -0,0 +1,27 @@
+import FullCalendar from "@fullcalendar/react";
+import interactionPlugin from "@fullcalendar/interaction";
+import timeGridPlugin from "@fullcalendar/timegrid";
+import dayGridPlugin from '@fullcalendar/daygrid';
+import Link from 'next/link';
+import { useRef } from "react";
+import { EventInstance } from "@fullcalendar/common";
+
+const Calendar = () => {
+  /*const calendarRef = useRef(null);*/
+  return (
+    <FullCalendar
+      /*innerRef={calendarRef}*/
```

Comment:
Remove any commented out code

Reply:
If these are important lines, then try to make them work before we merge this. If not, just delete. Part of the reason we enforce not committing "dead" code is so that in the future we aren't unsure if a comment can be deleted or not (for fear of it being important).